global.json

```
@@ -3,10 +3,10 @@
     "dotnet": "7.0.100-preview.7.22377.5",
     "runtimes": {
       "dotnet/x64": [
-        "6.0.8"
+        "7.0.0-preview.7"
```

Couldn't get it to work without this. Needs the same for x86 as well.

```suggestion
        "7.0.0-preview.7.22375.6"
```